aleph/views/xref_api.py

```diff
@@ -53,6 +61,13 @@ def index(collection_id):
     """
     get_index_collection(collection_id, request.authz.READ)
     result = XrefQuery.handle(request, collection_id=collection_id)
+    if get_flag("party", default=False):
```

Comment:
I'm not convinced we really want to call this ``party`` - that's a bit cryptic: party to what? The name makes sense in our internal discourse but it'll age really badly in 18 months.

Reply:
I'll rename it to the more sensibly boring `evaluation_mode`